### DISCUSSION TITLE
[Bug]在主页动态中由"投稿视频"转换到“全部”时，客户端崩溃

### DISCUSSION THREAD

**Discussion by SuanOrange** [ywmoyue/biliuwp-lite#603] [bug]
### Bug 描述

在主页动态中由"投稿视频"转换到“全部”时，客户端崩溃

### 操作步骤

```plain text
1.打开应用(版本为v4.6.30.0)
2.在首页点击“动态”，应用此时显示动态中的“全部”界面
3.切换到“投稿视频”或“专栏”或“追番追剧”界面
4.再次点击“全部”，应用卡死随即崩溃
```

### 期望结果

可以正常进行切换界面

### 截图

https://github.com/ywmoyue/biliuwp-lite/assets/67503405/01e084ef-2ee3-43d9-bf6a-6673918b23fa

### 应用来源

Github/其他Github镜像站

### 安装方式

脚本安装

### 系统版本

其它 (需注明)

### 其它系统版本

Windows 10 22H2 (Build 19045)

### 设备类型

笔记本

### 设备架构

x64

### 应用架构

x64

### 备注

有时应用只会卡死
由“追番追剧”，“专栏”界面切换到“全部”时也会卡死崩溃
有时在主页直接点击“动态”时会卡死崩溃
动态展示方式为“列表”时不会出现这个bug，切换为瀑布流会出现这个bug
二维码登录能复现，短信登录无法复现

### 代码贡献

我更擅长发现问题

**Comment by GD-Slime** [comment]
我这里没有复现此问题。能否发一下日志文件呢？ 确保accesskey/accesstoken/csrf=...里的内容已清除或已经被修改为{hasValue}再发出来。

**Reply by SuanOrange** [reply]
我检查了一下日志，发现了
|StackTrace.UserDynamicStringToRichText|用户动态文本转富文本失败|The text associated with this error code could not be found.
[20240429.log](https://github.com/ywmoyue/biliuwp-lite/files/15144883/20240429.log)

**Reply by SuanOrange** [reply]
动态展示方式为“列表”时不会出现这个bug，切换为瀑布流会出现这个bug

**Reply by GD-Slime** [reply]
https://space.bilibili.com/1980489/dynamic 我测试了一下 此用户的动态即使为"列表"也会导致崩溃

**Comment by GD-Slime** [comment]
@ywmoyue 找到一位用户，其动态可以稳定复现此bug https://space.bilibili.com/1980489/dynamic
应该是此动态导致的：https://t.bilibili.com/772854364142305286 但不太清楚原因

**Reply by GD-Slime** [reply]
有意思的是这个人的动态也不能稳定复现，但3次中就会有一次崩溃

**Comment by ywmoyue** [comment]
我这边测试了下二维码登录能复现，短信登录无法复现

**Comment by ywmoyue** [comment]
应该是这个动态https://www.bilibili.com/opus/922046211927572500
中的链接中有个bv导致嵌套转链接然后出的问题
